CMakeLists.txt

```diff
@@ -57,4 +57,11 @@ find_package(Qt4 REQUIRED)
 include_directories(SYSTEM "${QT_INCLUDES}")
 add_definitions(${QT_DEFINITIONS})
 
+find_package(ITK 4.9)
+if(ITK-FOUND AND NOT ITK_WRAP_PYTHON)
```

Comment:
> Did you mean an underscore in ITK_FOUND?

Yes, thanks for checking that. I will add it to a follow-up.

> Also, can we test for the FFTW support as well?

Yes, ITK_USE_FFTWF and ITK_USE_FFTWD (for float and double) settings are exported by ITKConfig.cmake.  Are both needed?